src/main/java/ru/hh/jetty/GracefulServerConnector.java

```
@@ -1,129 +0,0 @@
-package ru.hh.jetty;
-
-import org.eclipse.jetty.io.ByteBufferPool;
-import org.eclipse.jetty.io.Connection;
-import org.eclipse.jetty.io.EndPoint;
-import org.eclipse.jetty.server.ConnectionFactory;
-import org.eclipse.jetty.server.HttpChannelState;
-import org.eclipse.jetty.server.HttpConnection;
-import org.eclipse.jetty.server.Server;
-import org.eclipse.jetty.server.ServerConnector;
-import org.eclipse.jetty.util.annotation.Name;
-import org.eclipse.jetty.util.ssl.SslContextFactory;
-import org.eclipse.jetty.util.thread.Scheduler;
-
-import java.util.Collection;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.Executor;
-import java.util.concurrent.Future;
-import java.util.concurrent.TimeUnit;
-import java.util.function.Supplier;
-
-public final class GracefulServerConnector extends ServerConnector {
```

На самом деле просто переименован в HHServerConnector с добавлением fail fast, но git не осилил.